.eslintrc

```diff
@@ -8,6 +8,7 @@
     "ecmaVersion": 2018
   },
   "rules": {
-    "no-console": 0
+    "no-console": 0,
+    "semi": ["error", "always"]
```

Comment:
Added https://github.com/medic/medic-builds-repo/issues/19 to change what we extend.